src/manifest.c

```diff
@@ -582,6 +582,16 @@ void subtract_manifests(struct manifest *m1, struct manifest *m2)
 			list1 = g_list_next(list1);
 			list2 = g_list_next(list2);
 
+			/* When both files are deleted, skip subtraction, since
```

Comment:
After thinking about that case some more, I determined that it's currently impossible for the last_change values to be equal in this case. Because both files would be present in the previous version and therefore subtracted, the delete for the current version would be present in the subtracted manifest but not in the manifest subtracted from.

I'll update the code comment for clarification, but no further test case is needed that I can think of right now.